### ISSUE TITLE
Support for showing disconnected devices / Removing stale devices

### ISSUE THREAD

**Issue by tgurr** [badaix/snapweb#7]
When making use of the `Play` button of the Snapweb web client a new client is created and they may end up piling up when using different browsers/devices/sessions so it would be great to be able to add a toggle to display disconnected devices and also have the ability to remove them because having to edit the server.json manually to get rid of them is quite cumbersome.

**Comment by badaix** [comment]
I would like to have a menu for this, similar to the one in Snapdroid, a button with three dots or a hamburger button that opens a context menu with an "about..." option and a "[x] show hidden clients" checkbox.
It's not there yet, because I'm by far not an expert in web development (this is my first web project), and without using a framework (not needed so far, will keep the site small), it's quite some fiddly task.
What you can do is replacing in line 278 of `snapcontrol.js`
```js
let hide_offline = true;
```
with 
```js
let hide_offline = false;
```
It should also show a button to remove the clients for each offline client

**Comment by radusuciu** [comment]
@badaix Seems like you're potentially interested in accepting PRs? If so, a few questions:
- are you interested in refactoring PRs?
- do you have any browser support goals - eg. IE11?

**Comment by badaix** [comment]
Of course, I'm accepting PRs. General design goals are to keep it small and simple and to run on relevant browsers, including iOS.

**Comment by novwhisky** [comment]
@badaix Are you against using any framework in this project? I see #32 proposed Vue, albeit with lots of extra changes. Something modern but lightweight could be beneficial having reviewed the source.

**Comment by badaix** [comment]
Fixed in [v0.6.0](https://github.com/badaix/snapweb/releases/tag/v0.6.0)
